core/simulations/configs/vaultLiquidation.ts

```diff
@@ -60,12 +61,23 @@ const simulation: Simulation = {
                 const liquidatedId = context.latestVaultId;
                 const vault = await fetchVault(TEST_NETWORK, liquidatedId);
                 await liquidateVault(TEST_NETWORK, vault.collateralType, vault.address);
+                return context;
             },
         },
         {
             title: 'Skip time',
             entry: async context => {
-                await warpTime(60, 60);
+                const contract = await getContract(
+                    TEST_NETWORK,
+                    getClipperNameByCollateralType(context.collateralType)
+                );
+                const auctionLifetime: number = (await contract.tail()).toNumber();
```

Comment:
Don't we have this number as a date already stored within collateral `auction.endDate` as per proposal in the issue? At least reuse the existing function https://github.com/sidestream-tech/unified-auctions-ui/blob/4693a9852b65ee87719d0b559aded9949386223f/core/src/fetch.ts#L11-L18 if you don't want to deal with dates

Reply:
> Don't we have this number as a date already stored within collateral auction.endDate

this is harder because we would have to fetch vault in the simulation step and add it to the context (or something like that)

Reply:
> Do you remember that this already exists or do you check every time?

I did a quick search for `.tail()` in the codebase